src/System.Diagnostics.Debug/tests/Configurations.props

```diff
@@ -2,8 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp-Unix;
-      netcoreapp-Windows_NT;
+      netstandard;
```

Comment:
Note that the .csproj file has DebuggerTests.cs with `Condition="'$(TargetGroup)' == 'netcoreapp'`. The tests in this file won't run anymore once you start building netstandard configuration only,